earn/src/components/markets/borrow/BorrowingWidget.tsx

```diff
@@ -277,72 +287,63 @@ export default function BorrowingWidget(props: BorrowingWidgetProps) {
+                      const uniswapPosition = account.assets.uniswapPositions.at(0);
+                      const collateral = account.assets.amount0.isGtZero() ? account.token0 : account.token1;
+                      const collateralAmount = collateral.equals(account.token0)
+                        ? account.assets.amount0
+                        : account.assets.amount1;
+                      const collateralColor = tokenColors.get(collateral.address);
+                      const ltvPercentage = computeLTV(account.iv, account.nSigma) * 100;
+                      return (
+                        <AvailableContainer
+                          $gradDirection='45deg'
+                          $gradColorA={collateralColor && rgba(collateralColor, 0.25)}
+                          $gradColorB={GREY_700}
+                          key={account.tokenId}
+                          onMouseEnter={() => setHoveredBorrower(account)}
+                          onMouseLeave={() => setHoveredBorrower(null)}
+                          className={account === hoveredBorrower ? 'active' : ''}
+                          onClick={() =>
+                            setSelectedBorrower({
+                              borrower: account,
+                              type: 'supply',
+                            })
+                          }
+                        >
+                          {uniswapPosition !== undefined ? (
+                            <div className='flex items-center gap-3'>
+                              <TokenIcons tokens={[account.token0, account.token1]} />
+                              <Display size='XS'>Uniswap Position</Display>
+                              <Display size='XXS' color={SECONDARY_COLOR}>
+                                {uniswapPosition.lower} ⇔ {uniswapPosition.upper}
+                              </Display>
+                            </div>
+                          ) : (
+                            <div className='flex items-center gap-3'>
+                              <TokenIcon token={collateral} />
+                              <Display size='XS'>
+                                {collateralAmount.toString(GNFormat.LOSSY_HUMAN)}&nbsp;&nbsp;{collateral.symbol}
+                              </Display>
+                            </div>
+                          )}
+                          <Display size='XXS'>{roundPercentage(ltvPercentage, 2)}%&nbsp;&nbsp;LLTV</Display>
+                        </AvailableContainer>
+                      );
+                    })}
                   </div>
                 </CardRow>
               </CardContainer>
             </CardWrapper>
             <div className='w-[52px] mt-[2px]'>
               <div className='w-[52px] h-[42px]' />
-              {borrowers &&
-                borrowers.map((borrower) => {
-                  const hasNoCollateral = borrower.assets.amount0.isZero() && borrower.assets.amount1.isZero();
```

Comment:
Functional difference here in that we will now check that Uniswap positions are non-zero. Is that good?

Reply:
I couldn't think of a good reason we had it the way we did. If you can think of one, lmk. My thought was that we wouldn't want to show the health if there is no borrower to show.